native/package.json

```diff
@@ -43,6 +43,7 @@
     "@sentry/react-native": "^1.7.2",
     "@testing-library/jest-native": "^3.4.2",
     "@testing-library/react-native": "^7.0.2",
+    "build-configs": "file:../build-configs",
```

Comment:
The correct command to do this is `yarn workspace web add build-configs@0.0.1`

```
    "@integreat-app/react-sticky-headroom": "^1.0.4",
    "build-configs": "0.0.1",
    "detect-browser": "^4.1.0",
```

Reply:
Actually, that does not work. Getting the same error again as previously that babel interop is not found. AFAICS this command only creates a symlink.

What does your snippet below mean?

Reply:
thats what yarn generates after running the command above. Yeah my command should generate a symlink. Isn't that what we want?

Reply:
Does not work with metro afaics, at least I couldn't get it to work correctly :/

Reply:
I created an issue: https://issues.tuerantuer.org/browse/IGAPP-366